.github/workflows/deploy.yml

```diff
@@ -0,0 +1,53 @@
+name: Deploy to Vercel
+
+on:
+  push:
```

Comment:
For demo purposes, deployment runs on every push. We replace with 
```yml
on:
  push:
    branches:
    - master
```
before production